llvm/lib/Target/AIE/AIE2AddrSpace.h

```diff
@@ -41,6 +44,73 @@ enum class AddressSpaces {
 enum class AIEBanks { A, B, C, D, TileMemory };
 
 } // end namespace AIE2
+
+class AIE2AddrSpaceInfo : public AIEBaseAddrSpaceInfo {
```

Comment:
In this a final class?

Reply:
yes, have added the "final" keyword there.